postgres/logical-replication/sql/cloudgov/4_create_heartbeat.sql

```diff
@@ -0,0 +1,15 @@
+--Creates the heartbeat process to keep the WAL files from piling up on the db server
+--See https://wolfman.dev/posts/pg-logical-heartbeats/ for a detailed explaination
+DROP TABLE IF EXISTS camdaux._heartbeat;
```

Comment:
Is this needed when we have the pg_stat_replication cron job?

Reply:
You are correct.  I've removed the heartbeat process and added notes/comments to indicate that the pg_stat_replication job serves two purposes and should not be disabled.